sdks/python/apache_beam/examples/complete/juliaset/juliaset/juliaset.py

```diff
@@ -107,11 +107,14 @@ def run(argv=None):  # pylint: disable=missing-docstring
     # Group each coordinate triplet by its x value, then write the coordinates
     # to the output file with an x-coordinate grouping per line.
     # pylint: disable=expression-not-assigned
```

Comment:
This comment makes more sense next to where the pipeline is constructed. That is an opinion, feel free to ignore.